.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+    tags-ignore:
+      - '**'
+    paths-ignore:
+      - '**/CHANGELOG.md'
+      - '**/package.json'
+  pull_request:
+  workflow_dispatch:
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set Node Version
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+      - name: Install dependencies
+        run: npm i
+
+      - name: Bootstrap
+        run: npm run bootstrap
+
+      - name: Generate docs
+        run: npm run build:docs
+
+      - name: Semantic Release
+        uses: cycjimmy/semantic-release-action@v2
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN || secrets.GITHUB_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
This will fail on PRs triggered from forks because forks do not have access to secrets other than `secrets.GITHUB_TOKEN`. You might want to add an `if` conditional to this step to only run on pushes to `master`.

Reply:
- Bypassed forks
- Added if condition to skip the job